document_tax_rounding/models/purchase.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+from openerp import models, api, fields
+
+
+class PurchaseOrder(models.Model):
+    _inherit = 'purchase.order'
+
+    rounding_method = fields.Selection(
+        [('round_per_line', 'Round per line'),
+         ('round_globally', 'Round globally'), ],
+        string='Tax calculation rounding method',
+        index=True,
+        default=lambda self: self._default_tax_rounding_method()
```

Comment:
We can just use,
default=lambda self: self.env.user.company_id.tax_calculation_rounding_method